src/client/src/apps/MainRoute/widgets/spotTile/components/Tile/TileBusinessLogic.test.ts

```diff
@@ -16,108 +11,11 @@ import { PriceMovementTypes } from '../../model/priceMovementTypes'
 import { CurrencyPair, ServiceConnectionStatus } from 'rt-types'
 import { TileViews } from '../../../workspace/workspaceHeader/index'
 
-// edit mode, should not format on the fly
-// https://regex101.com/r/MrSCRE/7
-const shouldNotFormatOnTheFly = [
-  '.',
-  '12,321.',
-  '4.5',
-  '5.',
-  '.6',
-  '12.',
-  '0.1',
-  '0.4',
-  '34,876.',
-  '34,876.8',
-  '12132432.',
-  '12,123,123,123.',
-  '0.1',
-  '0.4',
-  '4.5',
-  '5.',
-  '.6',
-  '12.',
-  '34,876.',
-  '34,876.8',
-  '0.0',
-]
-// should format on the fly
-const shouldFormatOnTheFly = [
-  '000',
-  '00',
-  ',',
-  ' ',
-  '1,,,.',
-  '0',
-  '2.82',
-  '12.87',
-  '34.56',
-  '34,876.89',
-  '6.78',
-  '103,234,876',
-  '100,000,000',
-  '123456',
-  '123 456',
-  ',000,000',
-  '00,000,000',
-  '0.00',
-  '.98765',
-  '.05',
-  '1.1M',
-  '12.3k',
-  '2.5m',
-  '.11',
-  '0.11',
-  'Infinity',
-  'NaN',
-  '01234',
-]
-
-test('isEditMode should be true', () => {
-  shouldNotFormatOnTheFly.forEach(v => {
-    console.log('shouldNotFormatOnTheFly', v)
-    expect(isEditMode(v)).toBe(true)
-  })
-})
-
-test('isEditMode should be false', () => {
-  shouldFormatOnTheFly.forEach(v => {
-    console.log('shouldFormatOnTheFly', v)
-    expect(isEditMode(v)).toBe(false)
-  })
-})
-
 // invalid trading values, should disable trading (Buy/Sell buttons)
 // https://regex101.com/r/OWDRCO/2
```

Comment:
Do we still need that comment?

Reply:
yep, it refers to the new code `const invalidTradingValues = [0, -0, Infinity, -Infinity, NaN]` on line 16